assets/js/components/Dashboard/PlaceRow.tsx

```diff
@@ -87,8 +87,7 @@ const SelectBoxToggle = ({
 interface PlaceRowProps {
   place: Place;
   eventKey?: string;
-  // eslint-disable-next-line @typescript-eslint/ban-types
-  onClick: Function;
+  onClick: (arg?: any) => void;
```

Comment:
From a quick read of how it's used, this should be `arg?: boolean`. I'm not sure what the arg actually represents if present, though; it's called with the opposite of the component's `checked` value.

Reply:
I remember this being a dilemma back in the day. The problem is that `onClick` is either a simple `(arg?: boolean) => void` or it is a react-bootstrap `EventHandler` (which is typed to be a `React.EventHandler<React.SyntheticEvent>`. I'm going to try splitting out the types and creating `PlaceRowProps` a little differently. Let me know what you think about it.